rosdep/python.yaml

```diff
@@ -3694,6 +3694,13 @@ python-pymavlink:
 python-pymodbus:
   debian: [python-pymodbus]
   ubuntu: [python-pymodbus]
+python-pymodbus-pip:
```

Comment:
One last change, since these are identical please [make the `python3-` key an alias to the `python-` key](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1).

```yaml
python-pymodbus-pip: &migrate_eol_2027_04_30_python3_pymodbus_pip
  debian:
    pip:
      packages: [pymodbus]
  ubuntu:
    pip:
      packages: [pymodbus]
...
python3-pymodbus-pip: *migrate_eol_2027_04_30_python3_pymodbus_pip

```

Reply:
Ok, will do! Quick question though: how do I determine the EOL? Or better, how did you determine it?

Reply:
> how do I determine the EOL? Or better, how did you determine it?

I know we're transitioning to Ubuntu Jammy for ROS Rolling at the moment, so I assumed both that it counts as a supported platform and that it has the furthest away EOL date of April 2027.